default.nix

```diff
@@ -7,7 +7,7 @@ rec {
     stdenv.mkDerivation {
       name = "arx";
       buildCommand = ''
-        ${haskellPackages.arx}/bin/arx tmpx ${archive} -o $out // ${startup}
+        ${(import <nixpkgs> {}).haskellPackages.arx}/bin/arx tmpx -rm! ${archive} -o $out // ${startup}
```

Comment:
id recommend against importing nixpkgs like this, it makes it difficult to pin the version in a pure manner
instead try something like `import (nixpkgs.path) {}`